rocAL/rocAL/include/readers/image/reader.h

```diff
@@ -66,15 +77,19 @@ struct ReaderConfig
     void set_loop(bool loop) { _loop = loop; }
     void set_meta_data_reader(std::shared_ptr<MetaDataReader> meta_data_reader) { _meta_data_reader = meta_data_reader; }
     void set_sequence_length(unsigned sequence_length) { _sequence_length = sequence_length; }
-    void set_frame_step(unsigned step) { _step = step; }
-    void set_frame_stride(unsigned stride) { _stride = stride; }
+    void set_frame_step(unsigned step) { _sequence_frame_step = step; }
+    void set_frame_stride(unsigned stride) { _sequence_frame_stride = stride; }
     size_t get_shard_count() { return _shard_count; }
     size_t get_shard_id() { return _shard_id; }
     size_t get_batch_size() { return _batch_count; }
     size_t get_sequence_length() { return _sequence_length; }
-    size_t get_frame_step() { return _step; }
-    size_t get_frame_stride() { return _stride; }
+    size_t get_frame_step() { return _sequence_frame_step; }
```

Comment:
The rest of the video functions are in #ifdef. 
Any reason why this is not handled that way?

Reply:
```suggestion
    size_t get_frame_stride() { return _sequence_frame_stride; }
```
These functions are common to video and sequence reader
But the video_properties is used only for video reader, hence only that part is defined under ROCAL_VIDEO